server/backend/migration/20231003000000.go

```diff
@@ -0,0 +1,193 @@
+	log "github.com/sirupsen/logrus"
+	"gorm.io/gorm"
+)
+
+type multiLanguageTags struct {
+	MultiLanguageTags []tag `json:"tags"`
+}
+type tag struct {
+	TagNameEnglish string `json:"tagNameEnglish"`
+	TagNameGerman  string `json:"tagNameGerman"`
+}
+
+type multiLanguageNameTags struct {
+	MultiLanguageNameTags []nameTag `json:"tags"`
+}
+type nameTag struct {
+	TagNameEnglish string   `json:"tagNameEnglish"`
+	TagNameGerman  string   `json:"tagNameGerman"`
+	NotIncluded    []string `json:"notincluded"`
+	CanBeIncluded  []string `json:"canbeincluded"`
+}
+
+//go:embed static_data
+var staticData embed.FS
+
+// migrate20231003000000
+// migrates the static data for the canteen rating system and adds the necessary cronjob entries
+func (m TumDBMigrator) migrate20231003000000() *gormigrate.Migration {
+	return &gormigrate.Migration{
+		ID: "20231003000000",
+		Migrate: func(tx *gorm.DB) error {
+			if err := tx.Delete(&model.DishNameTagOption{}, &model.CafeteriaRatingTagOption{}, &model.DishNameTagOptionIncluded{}, &model.DishNameTagOptionExcluded{}).Error; err != nil {
+				return err
+			}
+			setTagTable("static_data/dishRatingTags.json", tx, backend.DISH)
+			setTagTable("static_data/cafeteriaRatingTags.json", tx, backend.CAFETERIA)
+			setNameTagOptions(tx)
+			errRating := addEntriesForCronJob(tx, "averageRatingComputation", 300)
+			if errRating != nil {
+				return errRating
+			}
+			errDish := addEntriesForCronJob(tx, "dishNameDownload", 302400)
+			if errDish != nil {
+				return errDish
+			}
+			return nil
+		},
+	}
+}
+
+func addEntriesForCronJob(tx *gorm.DB, cronName string, interval int32) error {
+	return tx.Session(&gorm.Session{Logger: logger.Default.LogMode(logger.Silent)}).Create(&model.Crontab{
+		Interval: interval,
+		Type:     null.StringFrom(cronName),
+		LastRun:  0,
+	}).Error
+}
+
+/*
+Updates the list of dishtags.
+If a tag with the exact german and english name does not exist yet, it will be created.
+Old tags won't be removed to prevent problems with foreign keys.
+*/
+func setNameTagOptions(db *gorm.DB) {
+	tagsNames := generateNameTagListFromFile("static_data/dishNameTags.json")
+	for _, v := range tagsNames.MultiLanguageNameTags {
+		var parentId int64
+
+		parent := model.DishNameTagOption{
+			DE: v.TagNameGerman,
+			EN: v.TagNameEnglish,
+		}
+
+		if err := db.Model(&model.DishNameTagOption{}).Session(&gorm.Session{Logger: logger.Default.LogMode(logger.Silent)}).Create(&parent).Error; err != nil {
+			fields := log.Fields{"en": v.TagNameEnglish, "de": v.TagNameGerman}
+			log.WithError(err).WithFields(fields).Error("Error while creating tag")
+		}
+		parentId = parent.DishNameTagOption
+
+		addCanBeIncluded(parentId, db, v)
+		addNotIncluded(parentId, db, v)
+	}
+}
+
+func addNotIncluded(parentId int64, db *gorm.DB, v nameTag) {
+	for _, expression := range v.NotIncluded {
+		fields := log.Fields{"expression": expression, "parentId": parentId}
+		err := db.Model(&model.DishNameTagOptionExcluded{}).
```

Comment:
I don't think this is nessesary
```suggestion
		err := db.
```